cmd/utils/flags.go

```diff
@@ -966,6 +970,10 @@ func setHTTP(ctx *cli.Context, cfg *node.Config) {
 	if ctx.GlobalIsSet(HTTPPathPrefixFlag.Name) {
 		cfg.HTTPPathPrefix = ctx.GlobalString(HTTPPathPrefixFlag.Name)
 	}
+
+	if ctx.GlobalIsSet(AllowUnprotectedTxs.Name) {
```

Comment:
There is no reason to check if it's set or not. That's only needed if the default has some special meaning. We only care about the actual value of the flag, so pls just use `cfg.UnprotectedAllowed = ctx.GlobalBool(AllowUnprotectedTxs.Name)`

Reply:
Yes there is, when we simply override it here, then we will always overwrite the value we read out of the config.
See martins comments below